index.html

```diff
@@ -38,6 +39,203 @@ <h1 class="text1">
+    <li>2015</li>
+</ul>
+</div>
+
+<div class="selection">
+    <p>A daily selection of 
+        privately personalized 
+        reads; no accounts or 
+        sign-ups required.</p>
+</div>
+<div class="languages">
+    <ul>
+        <li><span>html</span></li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+    
+<div class="button">
+    <button type="button">See Project</button>
+</div>
+</section>
+<section class="grid-container">
+<div class="snapshot-image">
+    <img src="images/multypost.png" alt="">
+</div>
+<div class="tonicmain">
+    <p>Tonic</p>
+</div>
+<div class="listitems">
+<ul>
+    <li><span>CANOPY</span></li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>Back End Dev</li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>2015</li>
+</ul>
+</div>
+
+<div class="selection">
+    <p>A daily selection of 
+        privately personalized 
+        reads; no accounts or 
+        sign-ups required.</p>
+</div>
+<div class="languages">
+    <ul>
+        <li><span>html</span></li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+    
+<div class="button">
+    <button type="button">See Project</button>
+</div>
+</section>
+<section class="grid-container">
+<div class="tonicmain">
+    <p>About Myself</p>
+</div>
+<div class="selection">
+    <P>Hello I’m a software developer! 
+        I can help you build a product , 
+        feature or website Look through 
+        some of my work and experience! 
+        If you like what you see and have
+         a project you need coded, don’t 
+         hestiate to contact me.</P>
+</div>
+<div class="languages">
+    <a href="#">LET'S CONNECT</a>
+</div>
+<div class="icons">
+    <ul>
+        <li><a href="https://twitter.com"><img src="images/twitter.png" alt="twitter"></a></li>
+        <li><a href="https://Linkedin.com"><img src="images/Linkedin icon.png" alt="Linkedin"></a></li>
+        <li><a href="https://medium.com"><img src="images/medium.png" alt="medium"></a></li>
+        <li><a href="https://github.com"><img src="images/github.png" alt="github"></a></li>
+        <li><a href="https://angellist.com"><img src="images/angellist.png" alt="angellist"></a></li>
+    </ul>   
+</div>
+<div class="button">
+    <button type="button">Get my resume</button>
+</div>
+</section>
+  
+</section>
+<section class="grid-container">
+<div class="drowdown">
+    <h1>languages</h1>
+    <i class="fa fa-angle-down"></i>
+</div>
+<div class="javascript-icon">
+    <img src="images/js.png" alt="">
+    <h1>Javascript</h1>
+</div>
+<div class="html-icon">
+    <img src="images/html.png" alt="">
+    <h1>HTML</h1>
+</div>
+<div class="css-icon">
+    <img src="images/css.png" alt="">
+    <h1>CSS</h1>
+</div>
+<div class="framework">
+    <h1>languages</h1>
+    <i class="fa fa-angle-right"></i>
+</div>
+<div class="skills">
+    <h1>skills</h1>
+    <i class="fa fa-angle-right"></i>
+</div>
```

Comment:
- [ ] Kyndly use `ul` and `li` tags for this section. 

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`.
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`